pkg/result/processors/severity.go

```diff
@@ -64,29 +61,32 @@ func (p *Severity) Process(issues []result.Issue) ([]result.Issue, error) {
 		return issues, nil
 	}
 
-	return transformIssues(issues, func(issue *result.Issue) *result.Issue {
-		if issue.Severity != "" && !p.override {
-			return issue
-		}
+	return transformIssues(issues, p.transform), nil
+}
 
-		for _, rule := range p.rules {
-			rule := rule
+func (p *Severity) transform(issue *result.Issue) *result.Issue {
+	for _, rule := range p.rules {
+		rule := rule
```

Comment:
I guess this line is redundant.

Reply:
No because we are using go1.21 :wink:

Reply:
before my PR it was not but now yes, I removed it.